### ISSUE TITLE
Usage stats collection default behavior should not be overridden

### ISSUE THREAD

**Issue by daniel-beck** [jenkinsci/kubernetes-operator#912]
**Describe the bug**
Per https://jenkinsci.github.io/kubernetes-operator/docs/security/ :

> The list below describes all the default security setting configured by the Jenkins Operator:
> * disable usage stats - Jenkins usage stats submitting is disabled

While many users consider this desirable, disabling this by default harms the project's ability to understand how Jenkins is used. We're collecting very limited configuration data, and (slightly) more extensive JEP-214 telemetry trials are not collected indefinitely.

As Jenkins (core) enables this by default, this subproject should not override that default. (Offering the option to disable it, like Jenkins does as well, is fine.)

**To Reproduce**
n/a

**Additional information**
n/a

Add error logs about the problem here (operator logs and Kubernetes events).

**Comment by brokenpip3** [comment]
yep make sense to let the users to decide to opt-out, I will try to add in the next release. Right now I'm working on upgrading the operator framework and the go version (basically to reduce the tech debt that we have atm).
